apps/institution-service/cypress/e2e/institutionsFiltersAndSearch.cy.ts

```diff
@@ -185,4 +191,47 @@ describe("/institutions with filters and search", () => {
       },
     );
   });
+
+  it("includes institutions with inactive integrations when includeInactiveIntegrations is passed", () => {
+    getInstitutionsRequest({
+      aggregatorFilter: ["finicity"],
```

Comment:
I would make a test institution and filter by it's unique name so that this test will work long term